_studio/shared/umc/codec/jpeg_enc/src/umc_mjpeg_video_encoder.cpp

```diff
@@ -0,0 +1,690 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#include "umc_defs.h"
+#if defined (MFX_ENABLE_MJPEG_VIDEO_ENCODE)
+
+#include <string.h>
+#include "vm_debug.h"
```

Comment:
remove

Reply:
I have a question. In `umc_mjpeg_video_encoder.cpp` `vm_debug.h` is not needed, so it's clear. But it used in `jpegdec.cpp`. Should I change `VM_ASSERT` to `assert`? What to do with `vm_debug_trace1`?

Reply:
> Should I change VM_ASSERT to assert?
Yes.
> What to do with vm_debug_trace1?
Depends on what are they doing. If there is nothing important - better to remove. If there is something important replace with MFX_TRACE_*.